doc/workflow.md

```diff
@@ -0,0 +1,163 @@
+# Tracing CVs with Inkscape from publication for the echemdb.website
+
+The cyclic voltammogram we trace is provided in Figure 2a in the [publication example](publication_example.pdf) and looks as follows:
+
+[<img src="sample_data_2.png" width="450"/>](sample_data_2.png)
+
+## Step 1: pdf, bib and metadata file preparation
+
+**1: Create a folder with the pdf, where the folder name is:**
+
+`FirstAuthorName_Year_FirstTitleWord_FirstPageNr`
+
+For the example pdf this comes down to
+
+`mustermann_2021_svgdigitizer_1`
+
+The Page number should be the page number in the published pdf. Since the example is not published the page number is `1`. In other cases this could be `1021`.
```

Comment:
```suggestion
The page number should be the page number in the published pdf. Since the example is not published the page number is `1`. In other cases this could be `1021`.
```